tests/test-class-amp-meta-box.php

```diff
@@ -77,7 +77,7 @@ public function test_enqueue_admin_assets() {
 	 * @see AMP_Post_Meta_Box::enqueue_block_assets()
 	 */
 	public function test_enqueue_block_assets() {
-		if ( ! function_exists( 'wp_get_jed_locale_data' ) && ! function_exists( 'gutenberg_get_jed_locale_data' ) ) {
+		if ( ! function_exists( 'the_gutenberg_project' ) ) {
```

Comment:
Maybe it would be good to use a function here that's very likely going to stay even when Gutenberg gets merged? For example `register_block_type`. Thoughts?

Reply:
Good point. I'm changing that now.

Reply:
Thanks, your suggestion is applied with https://github.com/Automattic/amp-wp/pull/1634/commits/3915ba24fd92b702b5d78895faf61c5f1e46f964